akka-docs/src/main/paradox/typed/testing.md

```diff
@@ -166,6 +166,12 @@ Scala
 Java
 :  @@snip [AsyncTestingExampleTest.java](/akka-actor-testkit-typed/src/test/java/jdocs/akka/actor/testkit/typed/javadsl/JunitIntegrationExampleTest.java) { #junit-integration }
 
+### Configuration
+
+By default the `ActorTestKit` loads configuration from `application-test.conf` if that exists, otherwise
+it is using default configuration (from reference.conf, application.conf not used).
+A specific configuration can be given as parameter when creating the TestKit.
```

Comment:
Extend this to mention how to load `application.conf` would be good, I think that is pretty common.

Reply:
will do

Reply:
added in 493c7a6